examples/robust_imports/src/types/azle_wrapper.ts

```diff
@@ -0,0 +1,34 @@
+export * as wrappedAzle from 'azle';
+import * as azle from 'azle';
+import * as deep from './deep/deep';
+export * from 'azle';
+// export * from './confusion';
+
+// export type text = number;
+
+// TODO make a test for
+// export * from 'azle';
+// export * from './confusion';
+// export type text = number;
+// where azle and confusion export conflicting declarations
+// where azle and confusion export conflicting declarations but the local export overrides it
+// where azle and confusion doesn't export conflicting declarations and there is a local export that overrides something from both of them
+// where azle and confusion and local don't conflict of override
+// TODO explore not doing our processes if the ts compiler would fail
+// TODO explore things that have multiple declarations
```

Comment:
Create tickets for these and remove the comments. Make sure to put them all in the same epic though....

Consider removing the whole file and saving for the bonus/extra credit tickets. Or at least give some sort of indicator that it isn't used.